pkg/backup/backup/backup_manager.go

```diff
@@ -263,12 +263,17 @@ func (bm *backupManager) makeExportJob(backup *v1alpha1.Backup) (*batchv1.Job, s
 	if backup.Spec.ServiceAccount != "" {
 		serviceAccount = backup.Spec.ServiceAccount
 	}
-	backupLabel := label.NewBackup().Instance(backup.GetInstanceName()).BackupJob().Backup(name)
+
```

Comment:
Also need to update `pkg/backup/backup/backup_cleaner.go`.

Reply:
https://github.com/pingcap/tidb-operator/pull/3892/commits/99bc01606643bf698ac0f363414a1a6fd00f7625